packages/route-viewer-overlay/src/index.js

```diff
@@ -16,6 +16,9 @@ const isGeomComplete = routeData => {
   );
 };
 
+/**
+ * An overlay that will display all polylines of the patterns of a route.
```

Comment:
Minor nit pick: isn't this overlay used for patterns from one or many routes?

Reply:
It's just for one route.

Reply:
Oh, yea, you're totally right. Thanks for confirming.